src/build.rs

```diff
@@ -183,4 +183,9 @@ fn main() {
     if cfg!(feature = "pkg-config-meta") {
         write_pkg_config();
     }
+
+    // MacOS: Allow cdylib to link with undefined symbols
+    if cfg!(target_os = "macos") {
```

Comment:
Could you put this above the `if cfg!(feature = "pkg-config-meta")` a few lines up? Just so the code that adds compiler options is grouped together to make reading easier.

Reply:
NP. Done.

Reply:
NP. Moved.